src/components/WebCLI/WebCLI.test.tsx

```diff
@@ -256,6 +258,7 @@ describe("WebCLI", () => {
         commands: ["status --color"],
       }),
     );
+    expect(props.onCommandSent).toBeCalledWith("status --color");
```

Comment:
This particular test isn't really about sending commands, it just happens to send one so it'd probably be better to add a new one for testing this case.